cmd/registry/cmd/resolve/resolve.go

```diff
@@ -120,5 +121,6 @@ func Command() *cobra.Command {
 
 	cmd.Flags().BoolVar(&dryRun, "dry-run", false, "if set, actions will only be printed and not executed")
 	cmd.Flags().IntVarP(&jobs, "jobs", "j", 10, "Number of actions to execute simultaneously")
+	cmd.Flags().IntVarP(&maxActions, "max-actions", "a", 100, "Maximum number of actions to execute")
```

Comment:
How does this differ from `--jobs`?

Reply:
`--jobs` limits the parallelism, the number of actions the controller can execute in parallel 
`--max-actions` is a cap on the total number of actions, as a whole the controller will execute only `max-actions` number of actions in one run. This to limit the controller from overloading the registry with large number of requests.